src/assets/i18n/en.json

```diff
@@ -449,6 +449,24 @@
       "vesselOperatorCarrierCode": "Operator Carrier Code",
       "size": "Vessel size (TEU)",
       "flag": "Vessel Flag",
+      "vesselType": {
+        "name": "vessel type",
+        "select": "Select a vessel type"
+      },
+      "dimensionUnit": {
+        "name": "Dimension unit",
+        "select": "Select a dimension unit"
+      },
+      "length": {
+        "name": "Length",   
```

Comment:
We call this "Length overall" in the IFS, so I think it should be `Length overall`.